### ISSUE TITLE
v1.3.0 fails to build on Fedora 35

### ISSUE THREAD

**Issue by junghans** [USCiLab/cereal#708]
From https://bugzilla.redhat.com/show_bug.cgi?id=1987402 / https://koji.fedoraproject.org/koji/taskinfo?taskID=72330774
```
[ 48%] Building CXX object sandbox/CMakeFiles/sandbox.dir/sandbox.cpp.o
cd /builddir/build/BUILD/cereal-1.3.0/redhat-linux-build/sandbox && /usr/bin/g++  -I/builddir/build/BUILD/cereal-1.3.0/./include -I/builddir/build/BUILD/cereal-1.3.0/sandbox/sandbox_shared_lib -Wall -g -Wextra -Wshadow -pedantic -Wold-style-cast -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -mbranch-protection=standard -fasynchronous-unwind-tables -fstack-clash-protection -std=gnu++11 -MD -MT sandbox/CMakeFiles/sandbox.dir/sandbox.cpp.o -MF CMakeFiles/sandbox.dir/sandbox.cpp.o.d -o CMakeFiles/sandbox.dir/sandbox.cpp.o -c /builddir/build/BUILD/cereal-1.3.0/sandbox/sandbox.cpp
In file included from /builddir/build/BUILD/cereal-1.3.0/unittests/common.hpp:59,
                 from /builddir/build/BUILD/cereal-1.3.0/unittests/pair.hpp:29,
                 from /builddir/build/BUILD/cereal-1.3.0/unittests/pair.cpp:28:
/builddir/build/BUILD/cereal-1.3.0/unittests/doctest.h:4021:47: error: size of array 'altStackMem' is not an integral constant-expression
 4021 |         static char             altStackMem[4 * SIGSTKSZ];
      |            
```

Build here: [build.log.txt](https://github.com/USCiLab/cereal/files/6903337/build.log.txt)

**Comment by kloczek** [comment]
Any update?

**Comment by junghans** [comment]
Updated the bundled version of doctree.h to a recent version helped (see #714).

**Comment by junghans** [comment]
Fixed in v1.3.1

**Comment by kloczek** [comment]
> Fixed in v1.3.1

Hmm .. last release it is 1.3.2 released almost 10 months ago.

**Comment by junghans** [comment]
> > Fixed in v1.3.1
> 
> Hmm .. last release it is 1.3.2 released almost 10 months ago.

Yup and the issue was actually fixed in #716 2 years ago.

**Comment by kloczek** [comment]
OK. Just reenabled in 1.3.2 test suite execution on packaging and looks like everything is OK now.
Thx 👍
